playbooks/bootstrap.yml

```diff
@@ -29,15 +29,22 @@
 #     debops bootstrap --sudo --limit host
 
 
-- name: Bootstrap host for Ansible management
+- name: Bootstrap Python support on a host
```

Comment:
"Python support" ? Shoudn’t that be "Prepare a host for Ansible management"?
https://github.com/drybjed/ansible-bootstrap/blob/0ebc30cef378f897668c05c0ff41f849091df1f0/docs/playbooks/bootstrap.yml